pyproject.toml

```diff
@@ -1,16 +1,76 @@
+# Reference: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#static-vs-dynamic-metadata
+
 [build-system]
```

Comment:
Does it matter where this is? Else I'd suggest putting `[project]` first.

Reply:
No, order does not matter, this is just the section order of the `pyproject.toml` reference. I've swapped them now.